vllm/engine/llm_engine.py

```diff
@@ -217,7 +217,7 @@ def _init_cache(self) -> None:
         # Since we use a shared centralized controller, we take the minimum
         # number of blocks across all workers to make sure all the memory
         # operators can be applied to all workers.
-        num_gpu_blocks = min(b[0] for b in num_blocks)
+        num_gpu_blocks = min(10500, min(b[0] for b in num_blocks))
```

Comment:
10500 is a max block count, since we take the minimum between it and whatever we get from the workers. I do not know why this extra value is introduced, and on 1xHPU the block count is actually 9250.